Libplanet/Net/Swarm.cs

```diff
@@ -759,6 +717,64 @@ private async Task BindingProxies()
             }
         }
 
+        private async Task DialToExistingPeers<T>(
+            BlockChain<T> blockChain,
+            CancellationToken cancellationToken)
+            where T : IAction, new()
+        {
+            long longestTipIndex = -1;
+            Peer longestPeer = null;
+
+            foreach (Peer peer in _peers.Keys)
+            {
+                try
+                {
+                    Pong pong = await DialPeerAsync(peer, cancellationToken);
+                    if (pong.TipIndex > longestTipIndex)
+                    {
+                        longestPeer = peer;
+                        longestTipIndex = pong.TipIndex.Value;
+                    }
```

Comment:
Instead of manually comparing the longest peer in the loop, how about making a simple array of pairs like `(Peer, long)[] peerLengths` and determining the longest peer after the loop ends like `(Peer longestPeer, long longestTipIndex) = peerLengths.Max(pair => pair.Item2);`?

Reply:
Interesting code. but I'm not sure that there is a big advantage enough to managing a separate array and looping it one more time.

Reply:
At first glance, it was hard to guess the intention, because code is not that short, and finding the peer who has the longest blockchain takes more lines than actual “dialing to existing peers” (as the method name says).  If we stick with the current version I suggest to write an explanation as a short comment.

Reply:
Okay, I'll leave comments and/or consider renaming `DialToExistingPeers()`.